src/ripple/overlay/impl/PeerImp.h

```diff
@@ -127,8 +128,8 @@ class PeerImp
     LedgerIndex maxLedger_ = 0;
     uint256 closedLedgerHash_;
     uint256 previousLedgerHash_;
-    std::list<uint256> recentLedgers_;
-    std::list<uint256> recentTxSets_;
+    std::deque<uint256> recentLedgers_;
```

Comment:
Do we have a policy somewhere as to when we use `std::deque` over `std::vector`?